src/index.js

```diff
@@ -1,39 +1,166 @@
+    if (taskitem) {
+      taskitem.innerHTML = '';
+    }
+    this.taskDtata.forEach((element) => {
+      const li = [];
+      li[element.index] = document.createElement('li');
+      li[element.index].setAttribute('id', element.index);
+      const p = [];
+      p[element.index] = document.createElement('p');
+      p[element.index].textContent = element.description;
+      p[element.index].setAttribute('id', element.index);
+      p[element.index].contentEditable = true;
+      const inputBox = [];
+      inputBox[element.index] = document.createElement('input');
+      inputBox[element.index].setAttribute('type', 'checkbox');
+      inputBox[element.index].classList.add('checkbox');
+      inputBox[element.index].setAttribute('id', element.index);
+      const button = [];
+      button[element.index] = document.createElement('button');
+      button[element.index].setAttribute('id', element.index);
+      button[element.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical icon"></i>';
+      li[element.index].append(inputBox[element.index], p[element.index], button[element.index]);
+      taskitem.append(li[element.index]);
+
+      p[element.index].addEventListener('click', (e) => {
+        e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+        e.target.nextSibling.style.cursor = 'pointer';
+
+        e.target.nextSibling.addEventListener('click', () => {
+          li[element.index].remove();
+          this.removetask(element.index);
+        });
+      });
+
+      li[element.index].addEventListener('mouseleave', (e) => {
+        button[element.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+        this.updatetask(e.target.id, e.target.innerText);
+      });
+      inputBox[element.index].addEventListener('change', (e) => {
+        const status = new TaskStatus();
+        if (e.target.checked === true) {
+          status.checked(this.taskDtata[element.index]);
+        } else {
+          status.unchecked(this.taskDtata[element.index]);
+        }
+        this.updatetask(e.target.nextSibling.id, e.target.nextSibling.innerText);
+      });
+      if (this.taskDtata[element.index].completed === true) {
+        inputBox[element.index].setAttribute('checked', 'checked');
+        li[element.index].classList.add('checked');
+        p[element.index].style.textDecoration = 'line-through';
+      } else if (this.taskDtata[element.index].completed === false) {
+        inputBox[element.index].removeAttribute('checked');
+        li[element.index].classList.remove('checked');
+        p[element.index].style.textDecoration = 'none';
+      }
+    });
+  }
+
+  addtask = (description, completed, index) => {
+    const newtask = new Tasks(description, completed, index);
+    this.taskDtata.push(newtask);
+    if (typeof window !== 'undefined') {
+      localStorage.setItem('LOCALLISTDB', JSON.stringify(this.taskDtata));
+    }
+  }
```

Comment:
As requested by the previous reviewer it's a project requirement to have this functionality of **Adding tasks** in a new file. 

## Solution:--

Then you can import that function in this file and use it as you want.

But obviously, for all variables declared here can't use them in that file so you can use the function parameters to pass them.